okhttp-testing-support/src/main/java/okhttp3/TestUtil.kt

```diff
@@ -59,20 +60,16 @@ object TestUtil {
     try {
       InetAddress.getByName("www.google.com")
     } catch (uhe: UnknownHostException) {
-      Assume.assumeNoException(uhe)
+      assumeNoException(uhe)
     }
   }
 
   @JvmStatic
   fun assumeNotWindows() {
```

Comment:
TIOLI: This might be more natural on PlatformRule?

Reply:
I think we probably want go from _Platform_ to the somewhat-independent parts it’s composed of, like TlsPlatform and VmPlatform. There’s an implied OsPlatform as a third thing that we’ve mostly avoided until now!

Reply:
+1 

The SocketAdaptor is also interesting because it allows for different socket types at the same time, and also cross cuts Android and JDK.